applications/FluidDynamicsApplication/python_scripts/compressible_slip_wall_process.py

```diff
@@ -0,0 +1,171 @@
+    long-term effect is less predictable but each step the momentum/velocity is
+    diminished by a factor of A^(-dt/period).
+
+    STEADY STATE
+    ------------
+    The node is given the SLIP flag, which signals the stategy that it is in
+    charge of applying the boundary condition.
+
+    Key parameters
+    --------------
+    - rampup_period: Duraation of the ramp-up stage.
+    - interval:  The time in this process runs (RAMP_UP + STEADY).
+    - recompute_normals: Enable computing the normals every step.
+    - variable: The variable to control, usually MOMENTUM of VELOCITY
+    """
+
+    decay_constant = 1000
+
+    # Stages
+    AWAITING = 0
+    RAMP_UP = 1
+    STEADY = 2
+    FINISHED = 3
+
+    def __init__(self, model, settings):
+        KratosMultiphysics.Process.__init__(self)
+
+        self.interval = KratosMultiphysics.IntervalUtility(settings)
+
+        settings.ValidateAndAssignDefaults(self.GetDefaultParameters())
+        modelpart_name = settings["model_part_name"].GetString()
+
+        self.model_part = model[modelpart_name]
+        self.rampup_period = settings["rampup_period"].GetDouble()
+        self.rampup_enabled = True
+        self.recompute_normals = settings["recompute_normals"].GetBool()
+        self._stage = self.AWAITING
+        self.variable = KratosMultiphysics.KratosGlobals.GetVariable(settings["variable_name"].GetString())
+
+    def GetDefaultParameters(self):
+        return KratosMultiphysics.Parameters(
+            """
+            {
+                "model_part_name"   : "SPECIFY_MODEL_PART_NAME",
+                "interval"   : [0, "End"],
+                "rampup_period"   : 0,
+                "recompute_normals" : false,
+                "variable_name" : "MOMENTUM"
+            }
+            """
+        )
+
+    def Check(self):
+        if self.rampup_period <= 0:
+            raise ValueError("Parameter rampup_period must be positive or zero.")
+
+        full_duration = self.interval.GetIntervalEnd() - self.interval.GetIntervalBegin()
+        if self.rampup_period > full_duration:
+            msg = "Parameter rampup_period must greater than the interval ({0} < {1})."\
+                .format(self.rampup_period, full_duration)
+            raise ValueError(msg)
+
+        self.model_part.HasNodalSolutionStepVariable(self.variable)
+
+        normal_type = type(KratosMultiphysics.NORMAL)
+        if not isinstance(self.variable, normal_type):
+            msg = "Variable must be a vector with the same size as NORMAL. Provided variable {} is not"\
+                .format(self.variable.Name())
+            raise TypeError(msg)
+
+    def ExecuteInitialize(self):
+        KratosMultiphysics.NormalCalculationUtils().CalculateNormals(self.model_part, True)
+        KratosMultiphysics.VariableUtils().SetFlag(KratosMultiphysics.SLIP, False, self.model_part.Nodes)
+
+        if self.rampup_period < 1e-18:
```

Comment:
```suggestion
        if not self.rampup_period > 0.0:
```
I think it is cleaner.